src/main/java/com/puppycrawl/tools/checkstyle/checks/metrics/NPathComplexityCheck.java

```diff
@@ -119,14 +119,58 @@
  * &lt;module name="NPathComplexity"/&gt;
  * </pre>
  * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * public void foo() { // OK, NPath complexity is less than default threshold
```

Comment:
Please add a violation case as well. E.g. use a `for` loop which violates default threshold.

Reply:
done

Reply:
Why did you add a violation case in the second example? It should belong to the first example.

Reply:
I see my mistake. done.